compair/api/__init__.py

```diff
@@ -322,6 +328,9 @@ def log_events(log):
     on_file_get.connect(log)
     on_file_delete.connect(log)
 
+    from .demo import on_user_demo_create
+    on_user_demo_create.connect(log)
+
```

Comment:
I've also moved this into its own function so that it isn't mixed in with the others (needed to do this because it wasn't easy to access the app config from this point since they are connected before the app is loaded)